_data/software.yaml

```
@@ -0,0 +1,45 @@
+- title: relentless
+  subtitle:
+  group: featured
+  image: images/photo.jpg
+  link: https://github.com/mphowardlab/relentless
```

Should we link to the documentation or the code? It would be nice if there were a way to link both in, not sure if this template supports that or if we could customize it that way. Maybe with like a little book logo for documentation and github logo for code?
```suggestion
  link: https://relentless.readthedocs.io
```